### DISCUSSION TITLE
Renaming Exports

### DISCUSSION THREAD

**Discussion by whynot2024** [light-and-ray/sd-webui-replacer#85] [Q&A]
is it possible to keep the file name of the original image and then maybe add suffix?

**Comment by light-and-ray** [comment]
It is for batch upload and from dir

**Comment by whynot2024** [comment]
Thanks for your reply! I was trying to keep the naming with suffix instead of prefix so that I can have the original and the export next to each other in file browser. I think batch upload keeps the name after the seed number. Is it possible to start with file name and then add everything after?

**Reply by light-and-ray** [reply]
possible, but I don't know why
